nilearn/_utils/glm.py

```diff
@@ -143,11 +143,41 @@ def _check_run_tables(run_imgs, tables_, tables_name):
     return tables_
 
 
-def z_score(pvalue):
-    """ Return the z-score corresponding to a given p-value.
+def z_score(pvalue, one_minus_pvalue=None):
+    """ Return the z-score(s) corresponding to
+    certain p-value(s) and cdfvalues(s) provided as inputs.
+
+    Parameters
+    ----------
+    pvalue: float or 1-d array shape=(n_pvalues,) computed using
+            the survival function
+
+    cdfvalue: float or 1-d array shape=(n_cdfvalues,), optional;
```

Comment:
the name of this parameter has changed to `one_minus_pvalue`?

Reply:
Yes, indeed. This has been fixed.